plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -410,10 +480,16 @@ fu_pxi_device_write_firmware (FuDevice *device,
 	if (!fu_pxi_device_fw_ota_init_new (self, g_bytes_get_size (fw), error))
 		return FALSE;
 
+	/* prepare write fw into device */
+	chunks = fu_chunk_array_new_from_bytes (fw, 0x0, 0x0, FU_PXI_DEVICE_OBJECT_SIZE_MAX);
+	if (!fu_pxi_device_check_support_resume (self, firmware, error)) {
```

Comment:
if you're using the function GError then you need to return FALSE and abort. I suspect what you actually want is a:

    g_autoptr(GError) error_local = NULL
    if (!fu_pxi_device_check_support_resume (self, firmware, &error_local)) {
        g_debug ("do not resume: %s", error_local->message);
        set stuff to 0...
    }

That way you don't steal the global GError and you can still report failure to resume in the debug log.